deployments/salt/splunk-otel-collector/fluentd.sls

```diff
@@ -0,0 +1,115 @@
+{% set splunk_fluentd_config = salt['pillar.get']('splunk-otel-collector:splunk_fluentd_config', '/etc/otel/collector/fluentd/fluent.conf') %}
```

Comment:
Please make sure that the td-agent service is restarted if the fluentd config file has changed.

Reply:
We have added fluentd_service.sls file for restarting td-agent service if the fluentd config file has changed.